tests/unit/test_parallel_login.py

```
@@ -30,3 +37,288 @@ def test_missing_name_returns_none(self):
+            p.start()
+        try:
+            # login.py calls sys.exit(1) at import time when config.py is missing (typical CI);
+            # mock.patch loads that module to patch login_with_retry, so sys.exit must be stubbed first.
+            with patch('sys.exit'), patch(
+                'packages.python.javdb_integrations.login.login_with_retry',
+                mock_login,
+            ), patch(
+                'packages.python.javdb_integrations.login.update_config_file',
+                return_value=False,
+            ):
+                success, cookie, proxy_name = session_mod.attempt_login_refresh(
```

Ruff (0.15.7)

[warning] Unpacked variable `cookie` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)